src/components/DateTimePicker/Internal/ocpicker.module.scss

```diff
@@ -924,6 +929,11 @@
         &-placeholder {
             input {
                 color: $picker-input-placeholder-color;
+
+                &-disabled,
+                &[disabled] {
+                    @include placeholder($picker-input-foreground-color);
```

Comment:
are we doing anything else visually for disabled input? if its just this it seems insufficient as it will look identical to an empty input

Reply:
The opacity dips to 0.3

Reply:
Roger.